process/modules/fin_to_conv.py

```diff
@@ -0,0 +1,121 @@
+from collections import defaultdict
+from typing import List
+import stanza
+import udon2
+from stanza.utils.conll import CoNLL
+import pyconll
+import pymorphy2
+from process.pipeline import ParaphrasePipeline
+from process.module import ParaphraseModule
+import re
+from process.preprocessing_utils import PreprocessingUtils
+
```

Comment:
PEP-8:

> Импорты должны быть сгруппированы в следующем порядке:
> - импорты из стандартной библиотеки
> - импорты сторонних библиотек
> - импорты модулей текущего проект

Две строки после импортов.

```suggestion
from collections import defaultdict
import re
from typing import List

import pyconll
import pymorphy2
import stanza
from stanza.utils.conll import CoNLL
import udon2

from process.module import ParaphraseModule
from process.preprocessing_utils import PreprocessingUtils


```